pkg/globalnet/main.go

```diff
@@ -86,8 +87,14 @@ func main() {
 	dynClient, err := dynamic.NewForConfig(cfg)
 	logger.FatalOnError(err, "Unable to create dynamic client")
 
-	// Set packetfilter driver to iptables. Once nftables is available, we'll check which driver is supported.
-	packetfilter.SetNewDriverFn(iptables.New)
+	// set up signals so we handle the first shutdown signal gracefully
+	ctx := signals.SetupSignalHandler()
+
+	globalConfigMap, err := configmap.Get(ctx, resource.ForConfigMap(k8sClient, spec.Namespace), configmap.Global)
+	logger.FatalOnError(err, "Error retrieving the global ConfigMap")
```

Comment:
QQ: With Admiral recent change, we should fatal error here only if  we couldn't read resource from k8s-api (if global cm doesn't exist, we should get empty cm) , right ?

Reply:
yeah `configmap.Get ` only returns an error in that case.